Makefile

```diff
@@ -288,52 +288,52 @@ react-init:
 .PHONY: react-build
 # React build.
 react-build:
-	cd frontend/ ; yarn run build
+	cd frontend/ ; yarn workspaces foreach --all run build
 	rsync -av --delete --delete-excluded --exclude=reports \
 		frontend/app/build/ lib/streamlit/static/
 
 .PHONY: frontend-fast
 # Build frontend into static files faster by setting BUILD_AS_FAST_AS_POSSIBLE=true flag, which disables eslint and typechecking.
 frontend-fast:
-	cd frontend/ ; yarn run buildFast
+	cd frontend/ ; yarn workspace @streamlit/app buildFast
```

Comment:
Could you please elaborate on why this is now only running for the `app` workspace?

Reply:
It actually does it already. I just simplified it to clean up the scripts in the main frontend package.json https://github.com/streamlit/streamlit/blob/develop/frontend/package.json#L13